### PR TITLE
[IMP] mail: open chat window when a message is received

### PR DESCRIPTION
**Pull request by tsm-odoo**

### CODE REVIEW

File: addons/mail/static/tests/new/messaging/messaging_tests.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                "id",
+                "=",
+                pyEnv["mail.message"].create({
+                    author_id: resPartnerId,
+                    body: "You're a wizard, harry.",
+                    model: "mail.channel",
+                    res_id: mailChannelId,
+                }),
+            ],
+        ]);
```
Comment (seb-odoo) [inline addons/mail/static/tests/new/messaging/messaging_tests.js]: would probably gain a lot of lines out of prettier by saving the id of new message in a const

Comment (tsm-odoo) [reply addons/mail/static/tests/new/messaging/messaging_tests.js]: 10 lines, done :+1:

File: addons/mail/static/tests/new/messaging/messaging_tests.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        });
+        const [message] = pyEnv["mail.message"].searchRead([["id", "=", messageId]]);
+        message.author = { name: message.author_id[1], id: message.author_id[0] };
+        await start();
+        await afterNextRender(() =>
+            pyEnv["bus.bus"]._sendone("channel-1", "mail.channel/new_message", {
+                id: message.id,
+                message,
+            })
+        );
```
Comment (seb-odoo) [inline addons/mail/static/tests/new/messaging/messaging_tests.js]: actually all of this could be `/mail/chat_post` rpc with `mockedUserId` param

Comment (tsm-odoo) [reply addons/mail/static/tests/new/messaging/messaging_tests.js]: updated :+1:

**Review by seb-odoo — APPROVED** [review]

**Pull request merged**